.github/workflows/ci_tests_dev.yaml

```
@@ -23,15 +23,15 @@ on:
     - cron: '0 0 * * *'
 
 jobs:
-  test_gmt_master:
+  test_gmt_main:
     name: ${{ matrix.os }} - GMT ${{ matrix.gmt_git_ref }}
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
         python-version: [3.9]
         os: [ubuntu-latest, macOS-11.0, windows-latest]
-        gmt_git_ref: [master]
+        gmt_git_ref: [main]
```

This references upstream GMT, so we have to keep it on 'master' for now.

```suggestion
        gmt_git_ref: [master]
```